scripts/pipeline/copy_data.pl

```diff
@@ -709,16 +686,28 @@ sub copy_genomic_align_blocks {
   my @copy_data_args = (undef, 'skip_disable_keys');
 
   #copy genomic_align_block table
-  if ($fix_gab or $fix_gab_gid) {
+  if ($fix_gab) {
     copy_data($from_dbc, $to_dbc,
        "genomic_align_block",
-       "SELECT genomic_align_block_id+$fix_gab, method_link_species_set_id, score, perc_id, length, group_id+$fix_gab_gid, level_id, direction".
+       "SELECT genomic_align_block_id+$fix_gab, method_link_species_set_id, score, perc_id, length, group_id, level_id, direction" .
```

Comment:
This is a vague and general formatting-related comment, but linked to both readability and potential future changes, and applies to the code prior to change too.
These longish queries could benefit from being written:
```
    my $sql = qq/
        SELECT 
            genomic_align_block_id + $fix_gab,
            method_link_species_set_id,
            score,
            perc_id,
            length,
            group_id,
            level_id,
            direction
        FROM
            genomic_align_block
        WHERE
            method_link_species_set_id = $mlss_id;
    /;
```
And then included in the `copy_data` code.

Reply:
I completely agree, but I tried to keep the formatting changes to a minimum because this file needs plenty! (starting from the fact it is 2 spaces instead of 4 :facepalm:) I can do this one though :+1:

Reply:
Yes, it is the fact that a lot of the formatting sucks that wasn't going to strongly suggest that one!